addons/mail/models/discuss/discuss_channel.py

```diff
@@ -160,6 +161,21 @@ def _constraint_group_id_channel(self):
 
     # COMPUTE / INVERSE
 
+    @api.depends("channel_member_ids", "auto_recompute_name")
+    def _compute_name(self):
+        for channel in self:
+            if channel.auto_recompute_name:
+                new_name = ", ".join(
+                    channel.channel_member_ids.mapped(
+                        lambda m: m.partner_id.name if m.partner_id else m.guest_id.name,
+                    )[:5],
+                )
+                if len(channel.channel_member_ids) > 5:
+                    new_name += "..."
+                if channel.name != new_name:
+                    channel.name = new_name
+                    self._bus_send_store(channel, ["name"])
```

Comment:
I don't think we should be sending on the bus inside a compute (especially since compute in views = change not necessarily saved).

Reply:
but if so, how can we sync the name with the client side? force `write()` is not allowed

Reply:
As I can see, compute here only modifies the in-memory cache, does not write to the database, does not trigger write(), and does not send any notifications and only triggers dependent inverse computations (may mark other computed fields as dirty).

and when I refresh the page, the name was correctly updated, means the value is being written into the database but not through .write()..... I will check with framework team

Reply:
It's done at flush time, not sure what you want but you may look at ``_compute_field_value(field_name)`` if you want to hook on the compute itself.